File: tests/fixtures/spring/fixed/LookupController.java
import java.sql.*;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class LookupController {
    private final Connection connection;
    public LookupController(Connection connection) { this.connection = connection; }

    @GetMapping("/lookup")
    public ResultSet lookup(@RequestParam String name) throws SQLException {
        PreparedStatement query = connection.prepareStatement("SELECT * FROM users WHERE name=?");
        query.setString(1, name);
        return query.executeQuery();
    }
}

File: tests/fixtures/csharp-classic/fixed/LookupController.cs
using System.Data.SqlClient;
using System.Web;
public class LookupController {
    public SqlDataReader Lookup(HttpRequest request, SqlConnection connection) {
        string name = request.QueryString["name"];
        var command = new SqlCommand("SELECT * FROM users WHERE name=@name", connection);
        command.Parameters.AddWithValue("@name", name);
        return command.ExecuteReader();
    }
}
